lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -225,14 +249,28 @@ def cert_action(module, iam, name, cpath, new_name, new_path, state,
             module.exit_json(changed=changed, msg='Certificate with the name %s already absent' % name)
 
 
+def read_files(cert, cert_body, key, key_body, cert_chain, cert_chain_body):
+    # if paths are provided rather than lookups read the files and return the contents
+    if not cert_body and os.path.exists(cert):
+        cert_body = open(cert, 'r').read().rstrip()
+    if not key_body and os.path.exists(key):
+        key_body = open(key, 'r').read().rstrip()
+    if not cert_chain_body and os.path.exists(cert_chain):
+        cert_chain_body = open(module.params.get('cert_chain'), 'r').read()
+    return cert_body, key_body, cert_chain_body
+
+
 def main():
     argument_spec = ec2_argument_spec()
     argument_spec.update(dict(
         state=dict(required=True, choices=['present', 'absent']),
         name=dict(),
         cert=dict(),
+        cert_body=dict(),
```

Comment:
I don't mind the idea of 2 params, but prefer the option that was discussed whereby the same param was used regardless.  If it was a path/file (os.path.isfile) then load the file, otherwise treat as string.
https://github.com/ansible/ansible/pull/20787
Not sure what is "normal" here currently though?